README.md

```diff
@@ -16,6 +16,8 @@ HoraeDB is a high-performance, distributed, cloud native time-series database.
 ## Contributing
 Any contribution is welcome!
 
+Discuss at [dev mailing list](mailto:dev-subscribe@horaedb.apache.org) ([subscribe](mailto:dev-subscribe@horaedb.apache.org?subject=(send%20this%20email%20to%20subscribe)) / [unsubscribe](mailto:dev-unsubscribe@horaedb.apache.org?subject=(send%20this%20email%20to%20unsubscribe)) / [archives](https://lists.apache.org/list.html?dev@horaedb.apache.org))
```

Comment:
```suggestion
Discuss at [dev mailing list](mailto:dev-subscribe@horaedb.apache.org) ([subscribe](mailto:dev-subscribe@horaedb.apache.org) / [unsubscribe](mailto:dev-unsubscribe@horaedb.apache.org) / [archives](https://lists.apache.org/list.html?dev@horaedb.apache.org))
```

Reply:
Maybe some clients don't support empty content, so non-empty content is used here.